back/db/migrate/20220203125126_create_news_posts.rb

```diff
@@ -0,0 +1,11 @@
+class CreateNewsPosts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :news_posts, id: :uuid do |t|
+      t.jsonb :title_multiloc
```

Comment:
It's often convenient to define default value and `not null` constraint. In this case, you don't need to handle `nil`, you can safely assume there's an object inside. 

```ruby
t.jsonb :title_multiloc, null: false, default: {}
```

We do similarly (not completely) in `back/db/migrate/20170318155729_create_ideas.rb`.